runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java

```diff
@@ -215,13 +218,23 @@ void rebuild()
 			priceTypeString = priceType == LootTrackerPriceType.HIGH_ALCHEMY ? "HA: " : "GE: ";
 		}
 
-		priceLabel.setText(priceTypeString + QuantityFormatter.quantityToStackSize(totalPrice) + " gp");
-		priceLabel.setToolTipText(QuantityFormatter.formatNumber(totalPrice) + " gp");
+		String averageString = kills > 1 ? QuantityFormatter.formatNumber(totalPrice / kills) + " gp" : "0";
+		if (showAverage)
+		{
+			priceLabel.setText(averageString);
+			priceLabel.setToolTipText(priceTypeString + QuantityFormatter.formatNumber(totalPrice) + " gp (total)");
+		}
+		else
+		{
+			priceLabel.setText(priceTypeString + QuantityFormatter.quantityToStackSize(totalPrice) + " gp");
+			priceLabel.setToolTipText("<html>" + QuantityFormatter.formatNumber(totalPrice) + " gp<br>" + averageString + " (average)</html>");			
```

Comment:
There's loads of trailing space here
```suggestion
			priceLabel.setToolTipText("<html>" + QuantityFormatter.formatNumber(totalPrice) + " gp<br>" + averageString + " (average)</html>");
```

Reply:
Sigh, sorry - missed that too.  Fixed.